lib/crypto/src/hash.rs

```diff
@@ -127,6 +127,16 @@ pub trait BuildHasher {
 }
 
 /// Hash the pair `(a, b)` with `state`.
+///
+/// # Arguments
+///
+/// * `a` - The first value to hash
+/// * `b` - The second value to hash 
+/// * `state` - The hasher state to use
+///
+/// # Returns
+///
+/// The finalized hash output from the hasher
```

Comment:
```suggestion
/// Hash the pair `(a, b)` with `state`.
///
/// Returns the finalized hash output from the hasher.
///
/// # Arguments
///
/// * `a` - The first value to hash
/// * `b` - The second value to hash 
/// * `state` - The hasher state to use
```
nit: current convention is we omit the `# Returns` section and include it in the general description.